t/op/sprintf2.t

```diff
@@ -1178,4 +1178,27 @@ if ($Config{intsize} == 4 && $Config{uvsize} > 4 && $Config{sizesize} > 4) {
     is($off2, 1, "offset after 0");
 }
 
+# %g formatting was broken on Ubuntu, Debian and perhaps other systems
+# for a long time. Here we verify that no such breakage still exists.
+# See https://github.com/Perl/perl5/issues/18170
```

Comment:
This should probably reference #18262 now.

Reply:
I think referring to the bug report is more correct here (which is what he did.)

Reply:
It may be more correct, but I actually would be happy to refer instead to the PR - simply because https://github.com/Perl/perl5/issues/18170 is such a long, shambolic account (my fault).
Also, this PR begins with a link to #18170 , so the original issue can easily be located.

So, I'm more than happy to proceed with the suggestion that @hvds made, but I won't do so unless given a clear go-ahead. (Personally, I'm not fussed either way.)
I've just now pushed a new APItest.pm (bumped version to 1.13) and APItest.xs (additional clarity) but I've left spritnf2.t as it already was.

Reply:
> Also, this PR begins with a link to #18170 , so the original issue can easily be located.

That was my reasoning too, but I'm happy to defer to @tonycoz .